cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -327,6 +331,25 @@ public void testCreateCluster(Params params, VertxTestContext context) {
                 emptyList()); //getInitialCertificates(getKafkaAssembly("foo").getMetadata().getName()));
     }
 
+    @ParameterizedTest
+    @MethodSource("data")
+    public void testCreateClusterWithJmxEnabled(Params params, VertxTestContext context) {
+        setFields(params);
+        Kafka kafka = getKafkaAssembly("foo");
+        kafka.getSpec().getKafka().setJmxOptions(new KafkaJmxOptionsBuilder()
+            .withAuthentication(new KafkaJmxOptionsAuthenticationPasswordBuilder().build())
+            .build());
+        createCluster(context, kafka,
+                Collections.singletonList(new SecretBuilder()
+                        .withNewMetadata()
+                        .withName(KafkaCluster.jmxSecretName("foo"))
+                        .withNamespace("test")
+                        .endMetadata()
+                        .withData(Collections.singletonMap("foo", "bar"))
+                        .build()
+                )); //getInitialCertificates(getKafkaAssembly("foo").getMetadata().getName()));
```

Comment:
commented line?

Reply:
I just followed how `testCreateCluster` called the method. I think we should keep this in so that if someone were to change `testCreateCluster` then they would know to change `testCreateClusterWithJmxEnabled`

Reply:
At this point, I don't remember why there is this commented line in the `testCreateCluster` method as well. @tombentley do you remember?

Reply:
I don't think we need to keep that comment.

Reply:
yes please